mapsforge-core/src/main/java/org/mapsforge/core/graphics/Paint.java

```diff
@@ -25,8 +26,40 @@ public interface Paint {
 
     int getTextHeight(String text);
 
+    default int getTextHeight(final String text, boolean includePadding) {
```

Comment:
The `default` methods in the interface are only used on Android.
So they can be hosted in `AndroidPaint` and avoid loading the interface.

Also, this is a feature of Java 8 and we're trying to be as backwards compatible as possible.

Reply:
We can transfer those to implementation classes, but then AWT will be left without label font padding concept. Maybe better to leave signatures in the interface and implement in both classes.

Reply:
We will have to choose one of 2 alternatives,
either implement and use the code on both platforms
or put the code in the platform class that is only used.